src/components/Contacts/ContactDetails/ContactDetails.tsx

```diff
@@ -50,11 +52,11 @@ export const ContactDetails: React.FC<Props> = ({
 }: Props) => {
   const { t } = useTranslation();
 
-  const [selectedTabIndex, setSelectedTabIndex] = React.useState(0);
+  const [selectedTabIndex, setSelectedTabIndex] = React.useState('0');
```

Comment:
Seemed like some of the typings for the tab components were expecting strings so I changed everything to a string.